src/rard/research/migrations/0071_add_duplicates_field_squashed.py

```diff
@@ -1,10 +1,14 @@
-# Generated by Django 3.2 on 2024-03-07 12:18
+# Generated by Django 3.2 on 2024-05-07 12:52
 
 from django.db import migrations, models
 
 
 class Migration(migrations.Migration):
 
+    replaces = [
+        ("research", "0071_add_duplicates_field"),
+    ]
+
```

Comment:
I don't think this will work. If I go to the development branch and run migrations I'm on 0071, then when I switch to this branch and run migrations nothing happens because it thinks its already run 0071. The only way to get it to work is to revert to 0070 while on the development branch, then switch to feature/duplicate-testimonia and run the migration. That won't work on production though where we've already merged the changes into the production branch.